.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,96 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches:
+      - master
+    types: [opened, synchronize]
+    paths-ignore:
+      - 'scripts/**'
+      - '**/*.md'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.18'
```

Comment:
Possible, this step is not required. In another repos like SDK, s3/http gateways in _Lint_ only two steps without Go installation:
* checkout
* run linter

Does this repo have some specific things, which required third step?

Reply:
We've had some problems in NeoGo with linter trying to check for 1.20+ features.